### PR TITLE
Fix return error in execute method

### PR DESCRIPTION
**Pull request by jf-viguier**
execute return a boolean.

| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Please be specific when describing the PR. What did you add, why did you submit it?
| Fixed ticket? | Fixes #{issue number here} if there is a related issue

### CODE REVIEW

File: src/content/1.7/development/database/db.md [hunk 1]
```diff
@@ -70,11 +70,11 @@ It prevent to loop in several arrays in order to get the first value of the firs
 <?php
 $request = "INSERT INTO `' . _DB_PREFIX_ . 'some_table` (`id_table`) VALUES (10)";
 
-/** @var array|false */
+/** @var bool $result */
```
Comment (PierreRambaud) [inline src/content/1.7/development/database/db.md:73]:
```suggestion
/** @var bool */
```

**Review by PierreRambaud — APPROVED** [review]

File: src/content/1.7/development/database/db.md [hunk 2]
```diff
@@ -70,11 +70,11 @@ It prevent to loop in several arrays in order to get the first value of the firs
 <?php
 $request = "INSERT INTO `' . _DB_PREFIX_ . 'some_table` (`id_table`) VALUES (10)";
 
-/** @var array|false */
+/** @var bool */
 $db->execute($request);
```
Comment (PierreRambaud) [inline src/content/1.7/development/database/db.md:74]:
```suggestion
/** @var bool $result */
$result = $db->execute($request);
```

**Review by PierreRambaud — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @jf-viguier

**Comment by jf-viguier** [comment]
You're welcome 👍 @Progi1984

**Pull request merged**